src/OVAL/probes/unix/linux/rpmverifyfile.c

```diff
@@ -345,10 +345,22 @@ void probe_fini (void *ptr)
 static int rpmverify_additem(probe_ctx *ctx, struct rpmverify_res *res)
 {
 	SEXP_t *item;
+	oval_schema_version_t oval_version;
+	const char *digest_attr_name;
+	const char *digest_attr_value;
 
 #define VF_RESULT(f) (res->oflags & (f) ? "not performed" : (res->vflags & RPMVERIFY_FAILURES ? "not performed" : (res->vflags & (f) ? "fail" : "pass")))
 #define FF_RESULT(f) (res->fflags & (f) ? true : false)
 
+	oval_version = probe_obj_get_platform_schema_version(probe_ctx_getobject(ctx));
+	if (oval_schema_version_cmp(oval_version, OVAL_SCHEMA_VERSION(5.11.1)) >= 0) {
+		digest_attr_name = "filedigest_differs";
+		digest_attr_value = VF_RESULT(RPMVERIFY_FILEDIGEST);
+	} else {
+		digest_attr_name = "md5_differs";
+		digest_attr_value = VF_RESULT(RPMVERIFY_MD5);
```

Comment:
That would require to add another test that will use content older versions of OVAL.